### ISSUE TITLE
Can't change useragent

### ISSUE THREAD

**Issue by OliverTrust** [arkenfox/user.js#1161]
Use this "profile" in selenium python and can't change useragent. Problem in this profile. With another profile changing work
`profile = webdriver.FirefoxProfile()`
`profile.set_preference('general.useragent.override', 'any_useragent')`
This perfectly work with default firefox profile. But with this user.js useragent all time `"Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:78.0) Gecko/20100101 Firefox/78.0"`
p.s.
`general.oscpu.override`
`general.platform.override`
`general.buildID.override`
not work too

**Comment by rusty-snake** [comment]
https://github.com/arkenfox/user.js/blob/9b8735a87a1696a0124246b92d5c6beadf2d4567/user.js#L1422-L1424
https://github.com/arkenfox/user.js/blob/9b8735a87a1696a0124246b92d5c6beadf2d4567/user.js#L1468-L1473

**Comment by Thorin-Oakenpants** [comment]
duplicate of `https://github.com/pyllyukko/user.js/issues/512`

Are you even using arkenfox?

This is the second person in 4 days, after #1158,  posting questions at pyllyukko, getting no answers, and then coming here.

[HERE](https://gitlab.torproject.org/tpo/applications/tor-browser/-/issues/27268#note_2653202) is your answer: from Tom Ritter (senior security dev at Mozilla and long time Tor guy), replying to my cleanup list of shit to remove (and those prefs **_are_** shit - they leak are practically useless)
